src/main/java/com/passbook/sparkeighteen/service/TransactionService.java

```diff
@@ -0,0 +1,48 @@
+package com.passbook.sparkeighteen.service;
+
+import com.passbook.sparkeighteen.peristence.POJO.TransactionRequest;
+import com.passbook.sparkeighteen.peristence.POJO.TransactionResponse;
+import com.passbook.sparkeighteen.peristence.entity.TransactionEntity;
+import com.passbook.sparkeighteen.peristence.entity.UserEntity;
+import com.passbook.sparkeighteen.peristence.repository.TransactionRepository;
+import com.passbook.sparkeighteen.peristence.repository.UserRepository;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class TransactionService {
+
+    private final TransactionRepository transactionRepository;
+    private final UserRepository userRepository;
+
+    public TransactionService(TransactionRepository transactionRepository, UserRepository userRepository) {
+        this.transactionRepository = transactionRepository;
+        this.userRepository = userRepository;
+    }
+
+    public TransactionResponse transact(Integer userID, TransactionRequest request) throws Exception {
+
+        Optional<UserEntity> optionalUser = userRepository.findById(userID);
+        if (optionalUser.isEmpty())
+            return TransactionResponse.builder().message("User not found. Please use a registered user").build();
+
+//        UserEntity user = optionalUser.get();
+
+        // TODO: Add deposit/withdraw logic and change below return accordingly
+
+        return TransactionResponse.builder().build();
+    }
+
+    private Float getZeroOrLastBalance(UserEntity user) {
```

Comment:
Remove this method too.

Reply:
Currently required to show closing balance